old-ui/app/components/network.js

```diff
@@ -60,6 +60,9 @@ Network.prototype.render = function () {
     } else if (providerName === 'poa' || parseInt(networkNumber) === 99) {
       displayName = 'POA Network'
       hoverText = ethNetProps.props.getNetworkDisplayName(networkNumber)
+    } else if (providerName === 'dai' || parseInt(networkNumber) === 100) {
+      displayName = 'Dai Chain'
```

Comment:
Shouldn't this use the `DAI_DISPLAY_NAME` defined in `network/enums.js`? The rest of the file seems to be doing the same (using strings instead of those constants) but maybe it should be changed.

Reply:
Yes, good idea. All dropdown display values are defined in `network/enums.js` now.